packages/mobile/src/identity/verification.ts

```diff
@@ -48,6 +26,29 @@ import { web3 } from 'src/web3/contracts'
 import { getConnectedAccount, getConnectedUnlockedAccount } from 'src/web3/saga'
 import { privateCommentKeySelector } from 'src/web3/selectors'
 
```

Comment:
Why the space here? I recommend letting VSCode organize the imports for you as the linter requires them to be alphabetized